classes/form/CustomerAddressForm.php

```diff
@@ -217,4 +209,83 @@ function (FormField $item) {
             'formFields' => $formFields,
         ];
     }
+
+    /**
+     * Performs validation on field values.
+     * Returns true if all field values are correct, false otherwise.
+     *
+     * @return bool
+     */
+    private function validateFieldsValues(): bool
+    {
+        $isValid = true;
+
+        $isValid &= $this->validatePostcode();
+        $isValid &= $this->validateField('firstname', 'isName', $this->translator->trans(
```

Comment:
@sowbiba why `Address->firstname` is not validated the same way as `Customer->firstname`? with `isCustomerName`?

Reply:
It's the way it was done in the objectModels
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Address.php#L129
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/Customer.php#L181 

But  isCustomer seems to be more restrictive about authorized charatcters so maybe better